src/charm.py

```diff
@@ -179,10 +180,20 @@ def _create_memory_storage(self, path: Path, size: int) -> None:
             else:
                 execute_command(["mount", "-o", f"remount,size={size}k", str(path)])
         except OSError as err:
-            logger.exception("Unable to create directory")
+            logger.exception("Unable to create storage directory")
+            # Ensure the path is not empty for next retry.
```

Comment:
Shouldn't it be `Ensure the path is empty for the next retry.` ?

Reply:
You are right